internal/pkg/cli/deploy.go

```diff
@@ -243,31 +249,43 @@ func (o *deployOpts) Run() error {
 		return err
 	}
 
-	if err := o.maybeInitWkld(); err != nil {
-		return err
+	for _, workload := range o.workloadNames {
+		if err := o.maybeInitWkld(workload); err != nil {
+			return err
+		}
+		deployCmd, err := o.loadWkldCmd(workload)
+		if err != nil {
+			return err
+		}
+		if err := deployCmd.Ask(); err != nil {
```

Comment:
Currently it appears as if this will force the user to wait for each subsequent workload initialization to run each subsequent `Ask()`. Is the intent to change this behavior when you parallelize the deployments?

Reply:
Yes, right now this is basic scaffold which will eventually be parallelized.